java/client/src/org/openqa/selenium/support/ui/CheckedFunction.java

```diff
@@ -0,0 +1,27 @@
+/*
```

Comment:
Fixed by the rebase, github just does not realized as this line has not changed.